packages/transport/src/modules/devices/DeviceSecretController.ts

```diff
@@ -61,22 +51,22 @@ export class DeviceSecretController extends TransportController {
         secret: DeviceSecretCredentials | CryptoExchangeKeypair | CryptoExchangePrivateKey | CryptoSignatureKeypair | CryptoSignaturePrivateKey | CryptoSecretKey,
         name: string
     ): Promise<SecretContainerCipher> {
-        const plainString: string = secret.serialize();
-        const plainBuffer: CoreBuffer = CoreBuffer.fromUtf8(plainString);
+        const plainString = secret.serialize();
+        const plainBuffer = CoreBuffer.fromUtf8(plainString);
 
-        const encryptionKey: CryptoSecretKey = await CoreCrypto.deriveKeyFromBase(this.getBaseKey(), 1, DeviceSecretController.secretContext);
+        const encryptionKey = await CoreCrypto.deriveKeyFromBase(this.getBaseKey(), 1, DeviceSecretController.secretContext);
 
-        const cipher: ICryptoCipher = await CoreCrypto.encrypt(plainBuffer, encryptionKey);
-        const date: CoreDate = CoreDate.utc();
-        const secretContainerInterface: ISecretContainerCipher = {
+        const cipher = await CoreCrypto.encrypt(plainBuffer, encryptionKey);
+        const date = CoreDate.utc();
+        const secretContainerInterface = {
```

Comment:
It may not be a problem, but when I hover the cursor over the `secretContainerInterface` variable, I no longer see that it is an `ISecretContainerCipher`.

Reply:
good catch!